canonical-web-and-design/hiring.canonical.com
# Add metrics page

## Done
- Added a page with metrics
## QA
- `dotrun`
- Go to http://localhost:8501/metrics
- Check tables are rendered
 ## Issue
Resolves #95
